spec/authorities/qa/authorities/iucat_libraries_spec.rb

```diff
@@ -5,6 +5,19 @@
   let(:matching_id) { 'B-WELLS' }
   let(:nonmatching_id) { 'foobar' }
 
+  context 'when configuration does not exist' do
+    describe '#all' do
+      let(:result) { authority.all }
+      it 'returns an empty Array' do
+        iucat_libraries_url = ESSI.config[:iucat_libraries]
```

Comment:
An alternative to the 3 lines for stashing, nullifying, and restoring the config value is:
`allow(ESSI.config).to receive(:[]).with(:iucat_libraries).and_return(nil)`
but I'm not sure which approach is more Hyrax-y

Reply:
@aploshay I pushed a new commit with the much improved test.